src/Mapper.js

```diff
@@ -2375,11 +2391,8 @@ export default Component.extend({
     const _opts = utils.pick(opts, ['existingOnly'])
     if (utils.isArray(record)) {
       const errors = record.map((_record) => schema.validate(_record, utils.pick(_opts, ['existingOnly'])))
-      const foundErrors = errors.filter((err) => err)
-      if (foundErrors.length) {
-        return errors
-      }
-      return undefined
+
+      return errors.some(Boolean) ? errors : undefined
```

Comment:
that's pity, the same story as in tests doesn't work here =) So, probably I need to change. But again this is only about IE8, does it make sense to care about it? https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/some
http://caniuse.com/usage-table

Let me know if you want to revert this back